script/redownload-electron-bins.js

```diff
@@ -12,8 +12,8 @@ if (typeof(downloadMksnapshotPath) !== 'undefined') {
   const path = require('path');
   const fs = require('fs');
 
-  const atomRepoPath = path.join('..', '..', '..', '..', 'atom', 'package.json');
-  const electronVersion = fs.existsSync(atomRepoPath) ? require(atomrepoPath).electronVersion : '6.1.12'
+  const atomRepoPath = path.join(__dirname, '..', '..', '..', '..', 'atom', 'package.json');
+  const electronVersion = fs.existsSync(atomRepoPath) ? require(atomRepoPath).electronVersion : '6.1.12'
```

Comment:
Off-topic, but it might make sense to call this variable `atomMetadata` or `atomMetadataPath` instead of `atomRepoPath`. That's more consistent with how Atom's scripts refer to `package.json` files (as "metadata").